### ISSUE TITLE
tidying up the repo so it can be used for other collections

### ISSUE THREAD

**Issue by evacougnon** [aodn/public-schema#66]
@mhidas , @leonardolaiolo , can this [file](https://github.com/aodn/public-schema/blob/master/product_index.csv) be moved to one of the existing folder? 
As we're about to merge a schema for a totally different product it will be great to keep the repo tidy.

Same to what's in `code` it's really specific to BGC/CPR data. Unless there is potential to use `code` for a more generic purpose.

Thanks

**Comment by leonardolaiolo** [comment]
I don't think we put to much thoughts about this repo being used by someone else..On the other hand, if that is the case for the future I agree with you Eva. Maybe keeping projects somehow enclosed in a single folder could be a solution, leaving only `LICENSE.md` and `README.md` common to all projects? @mhidas what do you think?
